### ISSUE TITLE
[Bug] The qbittorrent widget gets error if homepage is left open after a while

### ISSUE THREAD

**Issue by jmlc11** [benphelps/homepage#256]
**Describe the bug**
The qbittorrent widget gives API error after a while of being open.

**Expected behavior**
No error expected.

**Screenshots**
How it is when the page opens:
![image](https://user-images.githubusercontent.com/6006781/191991511-34918b74-7720-4b20-8da3-14951f1cd1c2.png)

How it gets after a moment:
![image](https://user-images.githubusercontent.com/6006781/191991467-9aacafd1-2515-4347-968c-456e1cd8323b.png)

**Configuration**
    - Qbittorrent:
        icon: qbittorrent.png
        href: https://urlforthis.org/
        widget:
          type: qbittorrent
          url: https://localip:8082
          username: user
          password: password
        server: my-docker # The docker server that was configured
        container: qbittorrent

**Additional context**
Docker deployed qbittorrent.

**Comment by JazzFisch** [comment]
How long a while is?  Seconds, minutes, hours?  Also can you reformat your configuration between ``` marks?  E.g.

````
```
- Qbittorrent
    icon: qbittorrent.png
    ...
```
````

**Comment by 1960697431** [comment]
same here

**Comment by jmlc11** [comment]
It's like 5 seconds.

**Comment by JazzFisch** [comment]
What are your docker logs for homepage?

**Comment by ShlomiPorush** [comment]
a temporary solution
https://github.com/benphelps/homepage/issues/221#issuecomment-1252075173

**Comment by jmlc11** [comment]
I have found what the issue was and it was my error:
I was using HTTPS instead of HTTP for the widget URL.
I first tried it with HTTP and it gave API error instantly, so i changed to HTTPS and got the widget like this:
https://user-images.githubusercontent.com/6006781/191991511-34918b74-7720-4b20-8da3-14951f1cd1c2.png
So i assumed it only worked with HTTPS, but today i checked the qbittorrent webUI config and HTTPS was not enabled.
So now with the URL in HTTP it works and looks like this:
![image](https://user-images.githubusercontent.com/6006781/192120841-404ea319-3589-4e1d-b682-298a592395e3.png)

So this Bug can be closed and sorry for the inconvenience.
